packages/edit-site/src/components/sidebar-navigation-screen/index.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	__experimentalNavigatorBackButton as NavigatorBackButton,
+	__experimentalNavigatorScreen as NavigatorScreen,
+} from '@wordpress/components';
+import { isRTL, __, sprintf } from '@wordpress/i18n';
+import { chevronRight, chevronLeft } from '@wordpress/icons';
+
+export default function SidebarNavigationScreen( {
+	path,
+	parentTitle,
+	title,
+	content,
+} ) {
+	return (
+		<NavigatorScreen
+			className="edit-site-sidebar-navigation-screen"
+			path={ path }
+		>
+			<VStack spacing={ 2 }>
+				<HStack
+					spacing={ 4 }
```

Comment:
Also magic numbers here. I guess I'm not familiar with these components enough to understand.

Reply:
It's just a random coefficient to the space between the components shown horizontally.